docker-compose.yml

```diff
@@ -1,11 +1,20 @@
-version: '2.1'
+version: "2.1"
 
 services:
+  firestore:
```

Comment:
Can we prepend the service names with "sds-" so that if a developer is working on another project they don't have to worry about conflicts when same the same docker images?